appd/request.py

```diff
@@ -766,10 +892,9 @@ def get_roles(self):
         :rtype: appd.model.Roles
         """
 
-        path = '/roles'
-        return self._rbac_v1_request_set(Roles, path)
+        return self._rbac_v1_request_set(Roles, '/roles')
 
-    def get_role(self, role_id):
+    def get_role_by_id(self, role_id):
```

Comment:
Why are we changing these method names? We should not break backward compatibility for users who are already using this API.